modules/acm/src/main/java/edu/gemini/epics/acm/CaApplySenderImpl.java

```diff
@@ -359,37 +381,45 @@ public State onTimeout() {
 
     private synchronized void onApplyValChange(Integer val) {
         if (val != null) {
+            State oldState = currentState;
             currentState = currentState.onApplyValChange(val);
             if (currentState.equals(IdleState) && timeoutFuture != null) {
                 timeoutFuture.cancel(true);
                 timeoutFuture = null;
             }
+            if(trace) LOG.debug("onApplyValChange(" + val + "): " + oldState.signature() + " -> " + currentState.signature());
```

Comment:
Wouldn't be better to disable them at the logging config?

Reply:
Yes, some day.